bundles/org.dataflowanalysis.analysis.converter/src/org/dataflowanalysis/analysis/converter/ProcessJSON.java

```diff
@@ -0,0 +1,366 @@
+package org.dataflowanalysis.analysis.converter;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.dataflowanalysis.analysis.converter.microsecend.*;
+import org.dataflowanalysis.analysis.converter.webdfd.*;
+import org.dataflowanalysis.dfd.datadictionary.*;
+import org.dataflowanalysis.dfd.dataflowdiagram.*;
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.emf.ecore.resource.Resource;
+import org.eclipse.emf.ecore.resource.ResourceSet;
+import org.eclipse.emf.ecore.resource.impl.ResourceImpl;
+import org.eclipse.emf.ecore.resource.impl.ResourceSetImpl;
+import org.eclipse.emf.ecore.xmi.XMLResource;
+import org.eclipse.emf.ecore.xmi.impl.XMLResourceFactoryImpl;
+
+public class ProcessJSON {
```

Comment:
This would also mean to move the WebDFD.json -> DFD.emf and DFD.emf -> WebDFD.json together into one class with would follow the locality principle of coherent code way better (although different technologies)